### ISSUE TITLE
Zenodo data bundle v0.5

### ISSUE THREAD

**Issue by nesnoj** [openego/eGon-data#602]
I'd like to have the [data bundle](https://zenodo.org/record/5743452) updated.

To be added (tick if added on NC):
- [x] eMobility trip data #553 @nesnoj 
- [x] RE potential areas (new data and drop the sandbox dependency) #595 @nesnoj 
- [ ] ~MaStR data (new data and drop the sandbox dependency) #519 @nesnoj~
- [ ] ~updated p-e-s output dataset (depends on #604 ) @ulfmueller~

Feel free to add further datasets..

**Comment by nesnoj** [comment]
@IlkaCu I have to do some further checks before the first three datasets can be included, so if @ulfmueller doesn't mind I'd like to postpone the update to next week.

**Comment by nesnoj** [comment]
No 1 and 2 are now updated on the NC, 3 will be added soon. I propose to make a release now especially to merge #624.

Do you want to add 4. now @ulfmueller or wait for 3.?

**Comment by nesnoj** [comment]
I moved 3 and 4 to a new issue #627

**Comment by IlkaCu** [comment]
Version 0.5 got published: https://zenodo.org/record/5846441
@nesnoj: You will update the deposit_id in your features branch, right?

**Comment by nesnoj** [comment]
Uh, obviously you mean the first.
Yes, I can change the data bundle's deposit id in my feature branches as I do not expect interferences with other datasets.
